### ISSUE TITLE
Левое меню навигации

### ISSUE THREAD

**Issue by OlegKolesnikoff** [bitrix-tools/b24-rest-docs#5]
Прыжок левого меню навигации наверх.

При выборе нужного раздела в левом меню навигации, меню пролистывается на самое верхнее положение, что приводит к тому, что нужно постоянно скролить его вниз, чтобы понять где находишься
![image](https://github.com/user-attachments/assets/5c71529f-18c6-443d-becb-39b566e082bd)

Так же в навигации не хватает 'хлебных крошек', которые были в старой версии
![image](https://github.com/user-attachments/assets/7f54e631-f394-497f-bcf0-40278779a3d5)

**Comment by serg-vostrikov-bitrix** [comment]
Да, знаем об этих недостатках и ждем обновления diplodoc, которые решат эти проблемы

**Comment by serg-vostrikov-bitrix** [comment]
навигацию в меню пофиксили

**Comment by Enotonoff** [comment]
> навигацию в меню пофиксили

Похоже, что не очень)

**Comment by serg-vostrikov-bitrix** [comment]
вероятно, старая js-либа, которая поиск реализует, торчит в кеше браузера
